includes/gateway/class-omise-payment-atome.php

```diff
@@ -145,10 +145,12 @@ private function validateMinRequiredAmount()
     public function charge($order_id, $order)
     {
         $currency = $order->get_currency();
-        $default_phone_selected = $_POST['omise_atome_phone_default'];
-        $phone_number = isset($default_phone_selected) && 1 == $default_phone_selected ?
+        $default_phone_selected = isset($_POST['omise_atome_phone_default']) ?
+            $_POST['omise_atome_phone_default']
+            : false;
+        $phone_number = true === (bool)$default_phone_selected ?
```

Comment:
Why do we need to compare with `true` if `(bool)` casts `$default_phone_selected` to a boolean?

Reply:
We don't need to compare. Thanks for noticing it. I have made the changes.